plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -91,21 +94,71 @@ public void testColumnName(String columnName)
     public void testRenameColumn()
     {
         // TODO (https://github.com/trinodb/trino/issues/7101) enable the test
+        // ClickHouse need resets all data in a column for specified column which to be renamed
         throw new SkipException("TODO: test not implemented yet");
     }
 
     @Override
     public void testDropColumn()
     {
-        // TODO (https://github.com/trinodb/trino/issues/7101) enable the test
-        throw new SkipException("TODO: test not implemented yet");
+        skipTestUnless(supportsCreateTable());
```

Comment:
Remove this since we now know we support create table. This clause makes sense for a BaseConnectorTest only.

Reply:
Same in other tests.